### PR TITLE
add doc for api keys

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+   GET /resource HTTP/1.1
+   Host: server.hopsworks.ai
+   Authorization: ApiKey <api_key>
+
+Generate an API key
+~~~~~~~~~~~~~~~~~~~~
+
+To generate an API key choose Settings from the top right dropdown
+menu and goto API keys tab. Give the new API key a unique name and
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/apiKeys.rst]: @ErmiasG `goto`

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   :alt: Create an API key
+   :figclass: align-center
+   :scale: 60%
+
+   Create new API key
+
+
+API key Security
+----------------
+Treat your secret API key as you would any other password:
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/apiKeys.rst]: @ErmiasG `would any` -> `would with any`

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
+=================
+API keys
+=================
+
+An API key allow a user or a program to make API calls
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: allows

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+without having to authenticate with username and password.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: with a username

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+To access an end-point using an API key a client should send the access token
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: endpoint .. key, a client

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+   GET /resource HTTP/1.1
+   Host: server.hopsworks.ai
+   Authorization: ApiKey <api_key>
+
+Generate an API key
+~~~~~~~~~~~~~~~~~~~~
+
+To generate an API key choose Settings from the top right dropdown
+menu and goto API keys tab. Give the new API key a unique name and
+add a scope to limit the access rights given to the key.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: "add a scope" is not concrete enough. 
add a scope to limit the access rights given to the key. Adding a scope means that you select the APIs that you want to allow this key to access - e.g., 'serving' and 'project' if this key will be used to access a model being served by Hopsworks.

File: docs/user_guide/hopsworks/apiKeys.rst (hunk 2 above)
Comment (jimdowling) [reply docs/user_guide/hopsworks/apiKeys.rst]: 'would any' is the more common way to express this in english.

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+   :scale: 60%
+
+   Create new API key
+
+
+API key Security
+----------------
+Treat your secret API key as you would any other password:
+
+1. Give API key the minimum required scope.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: Select the services for the API key scope.

File: docs/user_guide/hopsworks/apiKeys.rst [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+2. If an API key is stolen delete and recreate it.
```
Comment (jimdowling) [inline docs/user_guide/hopsworks/apiKeys.rst]: stolen, delete it and then recreate it.

**Comment by tkakantousis** [comment]
@ErmiasG JIRA number is required in the PR title

**Pull request merged**